### PR TITLE
Take "Should be able to add and remove and re-add multiple volumes out of quarantine"

### PR DESCRIPTION
**Pull request by awels**
Most failures are because of:
```
"message": "Error: container create failed: time=\"2021-07-03T13:40:59Z\" level=warning msg=\"exit status 1\"\ntime=\"2021-07-03T13:40:59Z\" level=error msg=\"container_linux.go:370: starting container process caused: process_linux.go:459: container init caused: rootfs_linux.go:59: mounting \\\"/var/lib/kubelet/pods/8aab0b65-20f4-4159-b498-8218ca1133e1/volumes/kubernetes.io~empty-dir/hotplug-disks\\\" to rootfs at \\\"/path\\\" caused: stat /var/lib/kubelet/pods/8aab0b65-20f4-4159-b498-8218ca1133e1/volumes/kubernetes.io~empty-dir/hotplug-disks: no such file or directory\"\n"
```
Which suggests we are running out of tempDir space. I am unable to reproduce locally which
is running serial. So seeing of making the test run serially will make a difference.

Signed-off-by: Alexander Wels <awels@redhat.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by brybacki** [comment]
so far looks good, lets try again: 
/test pull-kubevirt-e2e-k8s-1.20-sig-storage

File: tests/storage/hotplug.go [hunk 1]
```diff
@@ -383,6 +383,40 @@ var _ = SIGDescribe("Hotplug", func() {
...
+		var virtlauncher *corev1.Pod
+		for _, pod := range pods.Items {
+			if pod.ObjectMeta.DeletionTimestamp == nil {
+				virtlauncher = &pod
+				break
+			}
+		}
+		Expect(virtlauncher).ToNot(BeNil(), "Should find running virtlauncher pod")
+		Eventually(func() bool {
+			podList, err := virtClient.CoreV1().Pods(vmi.Namespace).List(context.Background(), metav1.ListOptions{})
```
Comment (enp0s3) [inline tests/storage/hotplug.go:404]: @awels I see that you've previously invoked the VMI namspace using `namespace := vmi.GetNamespace()`, why did you decide to directly read the namespace here using `vmi.Namespace`?

**Review by enp0s3 — CHANGES_REQUESTED** [review]
@awels Thank you for the fix! I have a small question below.

File: tests/storage/hotplug.go (hunk 1 above)
Comment (awels) [reply tests/storage/hotplug.go:404]: I probably copied the podList from somewhere else, but good point, I can just use the namespace I read already

**Review by enp0s3 — APPROVED** [review]
/approve

**Pull request merged**